src/main/java/wolox/training/models/User.java

```diff
@@ -0,0 +1,93 @@
+package wolox.training.models;
+
+import java.time.LocalDate;
+import java.util.Collections;
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import wolox.training.exceptions.BookAlreadyOwnedException;
+
+@Entity
+@Table(name = "users")
+public class User {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.AUTO)
+  private long id;
+
+  @Column(nullable = false)
+  private String username;
+  @Column(nullable = false)
+  private String name;
+  @Column(nullable = false)
+  private LocalDate birthDate;
+
+  @Column(nullable = false)
+
+  @ManyToMany(cascade = CascadeType.ALL)
+  @JoinTable(name = "book_user",
```

Comment:
It's unnecesarry to define a @JoinTable for @ManyToMany relationships, unless you want to give a custom name to the table and/or columns. Otherwise, JPA infers this automatically :)

Reply:
That's also good to know. Never trust tutorials blindly haha